SPECS/gpumon.spec

```diff
@@ -51,6 +51,9 @@ DESTDIR=%{buildroot} %{__make} install
 %{_unitdir}/xcp-rrdd-gpumon.service
 
 %changelog
+* Wed Jul 15 2020 Benjamin Reis <benjamin.reis@vates.fr> - 0.18.0-2
+- Mock gpumon to build without proprietary element
```

Comment:
Please also add "Related to https://github.com/xcp-ng/xcp/issues/381" both here and in the commit message.